### DISCUSSION TITLE
Account suspended with no response from GitHub support for over a week

### DISCUSSION THREAD

**Discussion by 84temp** [community/community#42339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My github account has been suspended with no warning from GitHub or reason with the message 'Access to your account has been suspended due to a violation of our Terms of Service' and I made a support ticket 10 days ago with no response from support. I understand this isn't the place for asking for support but I have no where else to post. So, I hope a GitHub employee can see this and help get my ticket seen or help me.

Account: github.com/84
Ticket ID: 1914959

**Comment by osawakoki** [comment]
Hello, @84temp 

I want to know how long it took to have your account unlocked and a detail about that.

**Reply by 84temp** [reply]
It hasn’t been unlocked nor have they replied to my ticket

**Comment by TCIano** [comment]
Hello, @84temp @84
I would also like to know how long it took for your account to be unlocked and the details about it.

**Reply by 84temp** [reply]
Never unlocked and support hasn’t replied to my ticket still
